### PR TITLE
Switch to arrow function as outlined in walkthr.

### PR DESCRIPTION
**Pull request by Ericas14**

### CODE REVIEW

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+0-// Copyright 2019 Google LLC
```
Comment (mkruskal-google) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js:1]: Revert this change

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (mkruskal-google) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: You shouldn't check in changes to the walkthrough files

Comment (Ericas14) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: In the walkthrough, it said, "Try changing the onclick attribute of the button to call the getRandomQuoteUsingArrowFunctions() function." Maybe it just meant to try it and change it back....?

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js (hunk 1 above)
Comment (Ericas14) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js:1]: Oops! Didn't know I changed that.

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html (hunk 2 above)
Comment (jbandlow-google) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: Making changes to the walkthrough files are fine, it's just preferred if you don't include those changes in your pull requests.

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js (hunk 1 above)
Comment (mkruskal-google) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js]: It happens :P
